### PR TITLE
Adding capability to untilt images

### PR DESCRIPTION
**Pull request by alexdesiqueira**
Untilts input images, based on EXIF parameters.

### CODE REVIEW

File: pipeline.py [hunk 1]
```diff
@@ -66,6 +68,45 @@ def create_layout(n_stages, plot_level):
                     ax_fourier, ax_tags]
 
 
+def untilt_image(image, image_path):
```
Comment (stefanv) [inline pipeline.py:71]: This API feels a bit strange; perhaps `untilt_image(image, metadata)`? That way, we can read the image along with its EXIF tags, and then have this function deal only with deciding what to do.

**Review by stefanv — APPROVED** [review]
Thanks, this looks good @alexdesiqueira.  I had a small suggestion about the API, but not critical.

File: pipeline.py (hunk 1 above)
Comment (alexdesiqueira) [reply pipeline.py]: Sure @stefanv. I divided this function into two; `read_orientation` reads the orientation, and `untilt_image` is run if necessary. For example, now the pipeline won't rotate the image if `orientation == 1`.

File: pipeline.py [hunk 2]
```diff
@@ -68,15 +68,42 @@ def create_layout(n_stages, plot_level):
...
     Parameters
     ----------
-    image_rgb : (M, N, 3) ndarray
-        RGB input image.
     image_path : str
         Path of the input image.
+    
+    Returns
+    -------
+    orientation : int or None
```
Comment (stefanv) [inline pipeline.py:81]: Docstring should describe what those integers mean. Are these angles now, in degrees?

File: pipeline.py [hunk 3]
```diff
@@ -87,24 +114,13 @@ def untilt_image(image, image_path):
...
-            angle = angles.get(orientation, 0)
-            return rotate(image, angle=angle, resize=True)
-        else:
-            print(f'Cannot evaluate orientation for {image_path}.')
-            return image
-    except ValueError:  # ... is not a valid TiffByteOrder
-        print(f'Cannot evaluate orientation for {image_path}.')
-        return image
+    # checking possible orientations for images.
+    angles = {1: 0,  # (top, left)
```
Comment (stefanv) [inline pipeline.py:118]: Do this translation inside the orientation reading function, then you can remove this utility function which only rotates.

**Comment by stefanv** [comment]
:rocket: Thanks!

**Pull request merged**